src/bp-forums/forums/template.php

```diff
@@ -2740,6 +2744,48 @@ function bbp_get_form_forum_visibility_dropdown( $args = '' ) {
 	return apply_filters( 'bbp_get_form_forum_type_dropdown', ob_get_clean(), $r );
 }
 
+/**
+ * Get group ID's for a child forum.
+ *
+ * @since BuddyBoss 1.7.6
+ *
+ * @param int $forum_id Forum id.
+ *
+ * @uses bbp_get_forum() Get forum.
+ *
+ * @return array
+ */
+function bb_get_child_forum_group_ids( $forum_id ) {
```

Comment:
move this function to the end of the file.

Reply:
Move the function at the end of the file.
https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/b2c663fb81f0630d02690c0ac5064d62ff9f8387